modules/reporting/app/views/cost_reports/index.html.erb

```diff
@@ -58,8 +58,8 @@ See docs/COPYRIGHT.rdoc for more details.
   <%= "<br />#{t(:information_restricted_depending_on_permission)}".html_safe unless User.current.admin?%>
 </p>
 
-<% content_for :sidebar do %>
-  <% [:private, :public].each do |type| %>
-    <%= render :partial => "report_category", :locals => { :report_type => type } %>
-  <% end %>
-<% end -%>
+<% if @project.blank? %>
```

Comment:
@ulferts Here we should also add the condition if there are any global queries like: Only show the global menu, if "@project.blank? && CostQuery.global.any?" to prevent an empty menu.
Do you know, how I can get these global queries?

Reply:
Or we could also also add a (or maybe two) menu entry for the global menu (`:application_menu`) to recreate the behaviour of the project menu and then get rid of using the sidebar.

Then, the check would be analogue to what we have for the project menu entry.